ashlar/migrations/0014_auto_20150929_1551.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.contrib.gis.db.models.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ashlar', '0013_auto_20150514_1712'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='record',
+            options={'ordering': ('-created',)},
+        ),
+        migrations.AlterField(
```

Comment:
I remember that we changed the srid a few times several weeks ago but that's it -- does it match what's in the models file? @kshepard , any thoughts?